docs/components/ColorsView.less

```diff
@@ -0,0 +1,183 @@
+@import (reference) "~less/utilities";
+
+.callout--box {
+  .padding--l;
+  border: @size_3xs solid @primary_blue;
+  background-color: @neutral_off_white;
+  height: 3rem;
+  &:hover {
+    .shade(border-color, @primary_blue, 2);
+  }
+}
+
+.color--block {
+  .display--block;
+  .margin--right--m;
+  .padding--top--5xl;
+  .padding--bottom--s;
+  .padding--x--m;
+  .borderRadius--topRight--xl;
+  width: 100%;
+  h4 {
+    .margin--none;
+  }
+}
+
+.large--color--block {
+  .margin--right--none;
+}
+
+.step--color--block {
+  .padding--top--2xl;
+  .margin--right--none;
+  .borderRadius--topRight--0;
+}
+
+// Primary colors:
+.bg--primary_blue {
+  background-color: @primary_blue;
+  color: @neutral_white;
+}
+
+.bg--primary_blue--tint-2 {
+  .tint(background-color,@primary_blue,2);
```

Comment:
add spaces between parameters after commas  (we should probably add a lint check for that eventually)